### PR TITLE
Add support for passing acr_values in auth requests in keycloak.js (#9383)

### LINKED ISSUE #9383: Support to customize acr_values query parameter in keycloak.js
### Description

Allow to customize the acr_values query parameter to let clients control the level of authentication they require.

### Discussion

_No response_

### Motivation

Given the newly added support for Step-Up Authentication  #7897, it makes sense to add support for configuring the required level of authentication in client libraries.

Currently it is hard to use the Step-Up Authentication support from clients since there is no explicit support for configuring acr_values for auth requests.

To play with the new feature we currently need to monkey patch the Keycloak object and augment the createLoginUrl function:
```javascript
...
let originalCreateLoginUrl = keycloak.createLoginUrl;

keycloak.createLoginUrl = function(options) {
    let url = originalCreateLoginUrl(options);
    url += "&acr_values=0";
    return url;
};
...
```

### Details

The keycloak.js adapter should allow to configure `acr_values` as query parameter for login and registration urls.

```javascript

let keycloak = new Keycloak({
    url: keycloakUrl,
    realm: realm,
    clientId: clientId
});

let initConfig = {
    onLoad: 'login-required', // redirects to login if not login
    // onLoad: 'check-sso', // shows login button of not logged in
    checkLoginIframe: true,
    checkLoginIframeInterval: 1,
    pkceMethod: 'S256',
    scope: scope,
    acrValues: "0"  // new parameter
};

...

keycloak.init(initConfig).success(onLoginSuccess);
```

The `acr_value` should be configurable via  `initOptions` and `loginOptions` for login and register urls.

### PR DESCRIPTION
**Pull request by thomasdarimont**
Added acrValues to login and init options.

Fixes #9383

### CODE REVIEW

File: docs/documentation/tests/src/test/resources/ignored-link-redirects [hunk 1]
```diff
@@ -15,4 +15,5 @@ https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshif
 https://access.redhat.com/login?redirectTo=https://access.redhat.com/terms-based-registry
 https://sso.redhat.com/auth/*
 REGEX:https://access.redhat.com/documentation/.*/index
+https://nodejs.org/en
```
Comment (jonkoops) [inline docs/documentation/tests/src/test/resources/ignored-link-redirects:18]: Why do we need to add this exception?

File: js/libs/keycloak-js/dist/keycloak.d.ts [hunk 2]
```diff
@@ -248,7 +255,14 @@ export interface KeycloakLoginOptions {
 	/**
 	 * Sets the `acr` claim of the ID token sent inside the `claims` parameter. See section 5.5.1 of the OIDC 1.0 specification.
 	 */
-	acr?: Acr;
+	acr?: Acr
```
Comment (jonkoops) [inline js/libs/keycloak-js/dist/keycloak.d.ts:258]: Looks like a semicolon was removed here.
```suggestion
	acr?: Acr;
```

**Review by jonkoops — CHANGES_REQUESTED** [review]
Couple of small nits, but otherwise LGTM

File: docs/documentation/tests/src/test/resources/ignored-link-redirects (hunk 1 above)
Comment (thomasdarimont) [reply docs/documentation/tests/src/test/resources/ignored-link-redirects:18]: I think this was due to an error during the documentation build that showed up when I did a full build.
I can try to remove it again.

Comment (jonkoops) [reply docs/documentation/tests/src/test/resources/ignored-link-redirects:18]: Yeah, if we can avoid it let's.

**Comment by thomasdarimont** [comment]
@jonkoops yay all tests passed :)

**Review by jonkoops — APPROVED** [review]
LGTM!

**Review by ssilvert — APPROVED** [review]
LGTM

**Pull request merged**